ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/Coverage/ClustersCoverageTable.tsx

```diff
@@ -16,23 +18,37 @@ import {
     Tooltip,
 } from '@patternfly/react-core';
 import { TableComposable, Thead, Tr, Th, Td, Tbody } from '@patternfly/react-table';
-import { SearchIcon } from '@patternfly/react-icons';
+import { CubesIcon } from '@patternfly/react-icons';
 
 import EmptyStateTemplate from 'Components/PatternFly/EmptyStateTemplate';
 import useRestQuery from 'hooks/useRestQuery';
 import useURLPagination from 'hooks/useURLPagination';
-import useURLSearch from 'hooks/useURLSearch';
-import { complianceEnhancedCoverageClustersPath } from 'routePaths';
+import {
+    complianceEnhancedCoverageClustersPath,
+    complianceEnhancedScanConfigsPath,
+} from 'routePaths';
 import { getAllClustersCombinedStats } from 'services/ComplianceEnhancedService';
 
+import ComplianceClusterStatus from '../ScanConfigs/components/ComplianceClusterStatus';
+import CoverageTableViewToggleGroup from './Components/CoverageTableViewToggleGroup';
+
 import {
     calculateCompliancePercentage,
     getCompliancePfClassName,
     getStatusCounts,
 } from './compliance.coverage.utils';
 
+const CreateScanConfigButton = () => {
+    return (
+        <Link to={complianceEnhancedScanConfigsPath}>
+            <Button variant="link" isInline>
+                Go to scan schedules
+            </Button>
```

Comment:
Can you see if link has same formatting even without `Button` element?

In the past, some integration tests failed because of element detached from the DOM when React re-renders a PatternFly `Button` element.

Reply:
By the way, `Create` in the name made me think about read versus write access level, but the path makes me think read is enough. Maybe the name is from copy-paste and might be more descriptive to this context?

Reply:
Oops good catch. I originally was going to have this go directly to the creation wizard which is why I named it this way but after testing that out, it felt a bit odd. I'll look into the Link/Button concern

Reply:
Link works by itself